modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,140 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Set an environment variable, such as `RELATED_IMAGE_MEMCACHED`, for any additional images used by the Operator in the `config/manager/manager.yaml` file:
++
+.Example `config/manager/manager.yaml`
+[%collapsible]
+====
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "docker.io/memcached:1.4.36-alpine"
+----
+====
+
+. Pull the related images with digests by replacing hard-coded image references with environment variables in the following files:
+
+* For Go-based Operator projects, update the `controllers/memcached_controller.go` file with the following changes:
++
+.`controllers/memcached_controller.go` diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+// deploymentForMemcached returns a memcached Deployment object
+...
+	Spec: corev1.PodSpec{
+        	Containers: []corev1.Container{{
+-			Image:   "memcached:1.4.36-alpine",
++			Image:   os.Getenv("RELATED_IMAGE_MEMCACHED"),
+			Name:    "memcached",
+			Command: []string{"memcached", "-m=64", "-o", "modern", "-v"},
+			Ports: []corev1.ContainerPort{{
+...
+----
++
+The `os.Getenv` function returns an empty string in the variable is not set. Set the environment variable before changing the file.
+====
 
-.. Update `spec.relatedImages`:
+* For Ansible-based Operator projects, update the `roles/memcached/tasks/main.yml` file with the following changes:
 +
+.`roles/memcached/tasks/main.yml` diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+spec:
+  containers:
+  - name: memcached
+    command:
+    - memcached
+    - -m=64
+    - -o
+    - modern
+    - -v
+-   image: "docker.io/memcached:1.4.36-alpine"
++   image: "{{ lookup('env', 'RELATED_IMAGE_MEMCACHED') }}"
+    ports:
+      - containerPort: 11211
+...
+----
+====
+
+* For Helm-based Operator projects, update the `helm-charts/memchached/values.yaml` file with the following changes:
++
+.`helm-charts/memchached/values.yaml` diff
+[%collapsible]
+====
+[source,diff]
+----
+  ## Memcached image and tag
+  ## ref: https://hub.docker.com/r/library/memcached/tags/
+  ##
+- image: memcached:1.5.20
++ image: "{{ lookup('env', 'RELATED_IMAGE_MEMCACHED') }}"
+...
+----
+====
+
+. Add the `BUNDLE_GEN_FLAGS` variable definition to your `Makefile` file and make the following changes:
```

Comment:
It seems like there is a lot of adding/editing in these yaml files. WDYT about adding in a line at like #1 or #2 or even dropping a line before the _operator requirements for supporting disconnected code_ that says in order to do this you will have to edit/add to yaml files.

Reply:
Also, I would remove "file" and just say:

> Add the `BUNDLE_GEN_FLAGS` variable definition to your `Makefile` and make the following changes: